src/main/scala/com/webtrends/harness/component/akkahttp/logging/AccessLog.scala

```diff
@@ -0,0 +1,58 @@
+package com.webtrends.harness.component.akkahttp.logging
+
+import akka.http.scaladsl.model.{DateTime, HttpMethod, StatusCode}
+import com.webtrends.harness.command.{BaseCommand, CommandBean}
+import com.webtrends.harness.component.akkahttp.AkkaHttpBase
+import com.webtrends.harness.component.akkahttp.AkkaHttpBase.TimeOfRequest
+import com.webtrends.harness.logging.Logger
+
+trait AccessLog  {
+  this: BaseCommand =>
+
+  val accessLog = Logger("AccessLog")
+  def logAccess(bean: CommandBean, statusCode: Option[StatusCode] = None) {
+
+    def reconstructPath(bean: CommandBean): String = {
```

Comment:
Do we want to just pass through the path so we don't have to bother reconstructing it every time?